v2/inc/c_logging/log_context_property_type_wchar_t_ptr.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef LOG_CONTEXT_PROPERTY_TYPE_WCHAR_T_PTR_H
+#define LOG_CONTEXT_PROPERTY_TYPE_WCHAR_T_PTR_H
+
+#include "c_logging/log_context_property_type_if.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+    int LOG_CONTEXT_PROPERTY_TYPE_INIT(wchar_t_ptr)(void* dst_value, size_t count, const wchar_t* format, ...);
+    int LOG_CONTEXT_PROPERTY_TYPE_GET_INIT_DATA_SIZE(wchar_t_ptr)(const wchar_t* format, ...);
```

Comment:
>wchar_t [](http://example.com/codeflow?start=72&length=7)

`wchar_t` in C comes from "some header" - which is missing as #include here.

Reply:
added #include <wchar.h>